bvm/ballerina-profiler/src/main/java/io/ballerina/runtime/profiler/Main.java

```diff
@@ -0,0 +1,310 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```
Check whether the added license headers matches with the above commented one.